sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -1098,4 +1098,24 @@ class SparkSqlAstBuilder extends AstBuilder {
 
     (ctx.LOCAL != null, finalStorage, Some(DDLUtils.HIVE_PROVIDER))
   }
+
```

Comment:
I have to put the parse process of `UnsetNamespacePropertiesCommand` here, because `UnsetNamespacePropertiesCommand` can only be defined within the module `sql/core`, and `AstBuilder` cannot access the class `UnsetNamespacePropertiesCommand` within the module